default/scripting/techs/construction/OUTPOST.focs.txt

```diff
@@ -5,6 +5,7 @@ Tech
     category = "CONSTRUCTION_CATEGORY"
     researchcost = 1
     researchturns = 1
+    unlock = Item type = Building name = "BLD_ABANDON_OUTPOST"
```

Comment:
this is malformed, needs to go after tags. breaks parsing.

Reply:
https://github.com/freeorion/freeorion/pull/3340

Reply:
https://github.com/freeorion/freeorion/commit/ce332ea9b5961fa21dd744238b627ab96ef81d91